### PR TITLE
Update cartoassets

### PR DESCRIPTION
**Pull request by nobuti**
This PR updates cartoassets to fix issues related to loader.

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1104,9 +1104,9 @@
       "resolved": "https://registry.npmjs.org/camshaft-reference/-/camshaft-reference-0.29.0.tgz"
     },
     "carto": {
-      "version": "0.15.1-cdb1",
+      "version": "0.15.1-cdb3",
```
Comment (nobuti) [inline npm-shrinkwrap.json]: @alonsogarciapablo is this ok?

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -1147,22 +1147,22 @@
     "cartodb-deep-insights.js": {
       "version": "0.0.2",
       "from": "cartodb/deep-insights.js#master",
-      "resolved": "git://github.com/cartodb/deep-insights.js.git#db4433f6bd036022add7f1ac4d7a3e9ec4536163",
+      "resolved": "git://github.com/cartodb/deep-insights.js.git#7f106bf59c969999fd635789b100b2a311a26d2b",
```
Comment (nobuti) [inline npm-shrinkwrap.json]: @donflopez is it safe to update this?

File: npm-shrinkwrap.json (hunk 1 above)
Comment (nobuti) [inline npm-shrinkwrap.json]: @rochoa ^^^

**Comment by nobuti** [comment]
@xavijam blocked we know if it's safe.

File: npm-shrinkwrap.json (hunk 1 above)
Comment (rochoa) [inline npm-shrinkwrap.json]: 👍 .

**Pull request closed without merging**